scripts/asm-installer/tests/run_basic_instance_template_creation

```diff
@@ -76,6 +76,10 @@ spec:
     serviceAccount: ${WORKLOAD_SERVICE_ACCOUNT}
 EOF
 
+  cleanup_old_instance_templates
```

Comment:
Can we make sure the cleanups all happen at the same step? e.g. https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/scripts/asm-installer/tests/setup_longterm_cluster#L34

It is a lot easier for debugging and understand what each step is doing.